co2calculator/distances.py

```diff
@@ -255,27 +274,20 @@ def geocoding_train_stations(loc_dict):
     # remove stations with no coordinates
     stations_df.dropna(subset=["latitude", "longitude"], inplace=True)
     countries_eu = stations_df["country"].unique()
```

Comment:
This could also be something we do once and store it in the sqlite db, so we don't need to do it at every function call?